docs/topics/api/overview.rst

```diff
@@ -246,6 +249,25 @@ specified.
 
 .. _api-versions-list:
 
+
+-----------
+Site Status
+-----------
+
+.. _`api-site-status`:
+
+This special endpoint just returns if the site is in read only mode, and if there is a site notice currently in effect.
```

Comment:
This kinda sounds like it'll be blocking if the site is not in read-only mode which isn't quite what you wanted to say I suppose. I wonder if simply removing the "just" makes it much clearer already.

Reply:
https://github.com/mozilla/addons-server/pull/12356/commits/4acd93a499ade786efc4967b45807f2550d4a248